tests/unit/components/sl-date-range-picker-test.js

```diff
@@ -0,0 +1,85 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-date-range-picker', 'Unit | Component | sl date range picker', {
+    needs: [
+        'component:sl-date-picker'
+    ],
+    unit: true
+});
+
+test( 'Default classNames are present', function( assert ) {
+    this.subject();
+    this.render();
+
+    assert.ok(
+        this.$().hasClass('sl-date-range-picker'),
+        'Default rendered component has class "sl-date-range-picker"'
+    );
+});
+
+test( 'Change focus to end date input upon start date change', function( assert ) {
+    this.subject();
+
+    let endDateInput = this.$( '.sl-daterange-end-date input' );
+
+    endDateInput.on( 'focus', () => {
+        assert.ok( true, 'End date intput recieves focus upon start date change');
+    });
+
+    this.render();
+
+    let startDateInput = this.$( '.sl-daterange-start-date input' );

```

Comment:
As this assignment is not used anywhere else in this test and because of fat arrow scope inclusion, just reference it directly on line 35.